### ISSUE TITLE
W[4]-Submission?

### ISSUE THREAD

**Issue by minhchau273** [npat1126/Twitter#2]
Hi Neal,

It looks like you have not created an issue for your submission. Please create an issue for all future project submissions following the steps in the submission guide.
Please update the README and be sure to also include an animated gif in your README to show a demo of your finished app. When properly linked, the gif should render (animate) within your README file. 

Your assignment needs to be updated within 24hrs in order for you to receive credit for this assignment.

Thanks,
Chau

**Comment by justnealpatel** [comment]
Yes. I asked one of my student leaders for an extension as my data was not
loading properly so I couldn't make a GIF from it. Thanks though. Although this is only chance to turn in a late assignment, just curious, would I have to alert you and my student leader? Thank you for the warning.

**Comment by justnealpatel** [comment]
/cc @codepathreview @minhchau273 I have uploaded my project. Unfortunately everything that could've gone wrong went wrong (Murphy's Law type of thing). I have everything working it's just that when I tried implementing the logout button, it doesn't go past the "Authorize" screen even if the login is successful, and just goes back to the login screen. This is what caused me to turn it in late, but then I fixed it in time to implement the favorite and retweet buttons and that worked. Hopefully you'll be able to find the bug if any. I tried seeking help on Slack, but I got no reply.
**EDIT**
I managed to fix my issue. It had to do with the AppDelegate. I hope you'll still accept this; it took me forever to find this bug.

**Comment by minhchau273** [comment]
:+1: Nice work! It's great you can figure out the problem yourself :smile: 

The point of this homework was to explore an example of a full MVC application with a RESTful API. 

### Some small comments:
- Nice work specifying the Auto Layout constraints.
- Nice work setting up the controls within the Tweet cells. It can sometimes be hard to have a clean design for embedded controls because it usually involves cumbersome plumbing. This is an instance where I think having the model handle the network action helps greatly simplify the implementation.

We have a detailed [Project 4 Feedback Guide](http://courses.codepath.com/snippets/ios_university/feedback_guides/project_4_feedback.md) which covers the best practices for implementing this assignment. Read through the feedback guide point-by-point to determine ways you might be able to improve your submission. You should consider going back and implementing these improvements as well. Keep in mind that one of the most important parts of iOS development is learning the correct patterns and conventions.

If you have any particular questions about the assignment or the feedback, feel free to reply here or email us at universitysupport@codepath.com.
